src/connector/src/sink/iceberg/mod.rs

```diff
@@ -307,19 +308,32 @@ impl IcebergConfig {
             iceberg_configs.insert(CATALOG_NAME.to_string(), self.catalog_name());
 
             if let Some(region) = &self.region {
+                // icelake
                 iceberg_configs.insert(
                     "iceberg.table.io.region".to_string(),
```

Comment:
Why need to keep the config for the deprecated `icelake` implementation?

Reply:
Icelake is still used in other crates like `frontend` (metadata) and `connector` (e.g. sink). The frontend part would be done in a later PR, but we still need icelake for sink.